lession13_dom_event/exercise-2-changeBackground/main.js

```diff
@@ -0,0 +1,15 @@
+const divChange = document.getElementById('divChange')
+let flag = 0
+const changeBg = divChange => {
+  if (flag === 0) {
+    divChange.style.background = 'green'
+    flag = 1 
+  }
+  else if (flag === 1) {
```

Comment:
Đặt if else kiểu gì đây?

```suggestion
  } else if (flag === 1) {
```